src/lab1/Main.java

```diff
@@ -0,0 +1,179 @@
+package lab1;
+
+import java.util.Scanner;
+
+public class Main {
+    static void showMenu() {
+        System.out.println("""
+                Press 1 to input movies.
+                Press 2 to display movies and ratings.
+                Press 3 to input more movies.
+                Press 4 to find statistics.
+                Press 5 to search for a movie.
+                Press 6 to update movie ratings.
+                Press 7 to delete a movie.
+                Press 8 to sort movies.
+                Press 0 to exit.
+                """);
+    }
+
+    static Movie[] addMovies(int n, Scanner sc, Movie[] movies) {
+        Movie[] newMovies = new Movie[movies.length + n];
+        System.arraycopy(movies, 0, newMovies, 0, movies.length);
+        for (int i = movies.length; i < newMovies.length; i++) {
+            sc.nextLine();
+            System.out.print("Enter movie name: ");
+            String movieName = sc.nextLine();
+            System.out.print("Enter movie rating: ");
+            double movieRating = sc.nextDouble();
+            Movie movie = new Movie(movieName, movieRating);
+            newMovies[i] = movie;
+        }
+        return newMovies;
+    }
+
+    static double findAvgRating(Movie[] movies) {
+        double avgRating = 0;
+        for (Movie movie : movies) {
+            avgRating += movie.getRating();
+        }
+        avgRating = avgRating / movies.length;
+        return avgRating;
+    }
+
+    static void moviesWithMaxAndMinRating(Movie[] movies) {
+        double maxRating = movies[0].getRating();
+        double minRating = movies[0].getRating();
+        for (Movie movie : movies) {
+            if (movie.getRating() > maxRating) {
+                maxRating = movie.getRating();
+            }
+            if (movie.getRating() < minRating) {
+                minRating = movie.getRating();
+            }
+        }
+        System.out.println("The movie with the maximum rating: " + maxRating);
+        System.out.println("The movie with the minimum rating: " + minRating);
+    }
+
+    static void findMovie(String movieName,Movie[] movies) {
+        for (Movie movie : movies) {
+            if (movie.getName().equals(movieName)) {
+                System.out.println("Rating of movie \"" + movieName + "\" is " + movie.getRating());
+                return;
+            }
+        }
```

Comment:
instead of printing finding movie here,you can just return index of movie, for reusability, write once use everywhere :)